ModularTegustation/tegu_jobs/chemre/chemresearcher.dm

```diff
@@ -0,0 +1,43 @@
+/datum/job/tegu/chemre
+	title = "Chemical Researcher"
+	department_head = list("Research Director")
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the research director, or the Chief medical officer, depending on which department you reside in."
+	selection_color = "#ffeeff"
+	exp_requirements = 60
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/chemre
+
+	access = list(ACCESS_MEDICAL, ACCESS_CHEMISTRY, ACCESS_RESEARCH, ACCESS_MINERAL_STOREROOM, ACCESS_RND)
```

Comment:
why do they need mineral storeroom access?
also later on we might wanna give the chem research labs their own access defines

Reply:
Mineral Storeroom to get things like silver for chemicals.

Reply:
They can get any material from medical techfab already. No need to give them access to ORM.